sql/timeseries/a11yScores.sql

```diff
@@ -1,24 +1,25 @@
 #standardSQL
-CREATE TEMPORARY FUNCTION getA11yScore(report STRING)
+# Lighthouse changed format of scores in v3.0.0 released in July 2018 so hande old and new
+CREATE TEMPORARY FUNCTION getA11yScore(reportCategories STRING, score STRING)
 RETURNS FLOAT64
```

Comment:
Since the function is called 5 times with the same inputs, I wonder if this would help reuse the results.
```suggestion
RETURNS FLOAT64 DETERMINISTIC
```

Reply:
Will test once we address other queries.

Reply:
Fixed in https://github.com/HTTPArchive/bigquery/pull/114/commits/8ddc5e6f138587b58c46bc7d991efa58d559b301

I do wonder if the caching of more complex JSON objects with potentially many different numbers could cause more memory issues than it saves in time! But lets give it a go.